spec/integration.rb

```diff
@@ -146,6 +146,7 @@
     s.replace_pattern /#{Dir.tmpdir}\/[\w-]+/io, 'TMPDIR'
     s.replace_pattern /\d{4}-\d\d-\d\d \d\d:\d\d:\d\d [-+]\d{4}/, '<#DATE#>'
     s.replace_pattern /\(Took \d+.\d+ seconds\)/, '(Took <#DURATION#> seconds)'
+    s.replace_pattern /\b#{Regexp.escape(Pod::VERSION)}\b/, '<#POD::VERSION#>'
```

Comment:
You'll also need to replace the cocoapods-downloader version, it's also in the output

Reply:
ah sure, fixing.